src/libraries/Microsoft.PowerFx.Core/Public/Values/RecordValue.cs

```diff
@@ -173,13 +201,6 @@ public virtual async Task<DValue<RecordValue>> UpdateFieldsAsync(RecordValue cha
             });
 
             return DValue<RecordValue>.Of(errorValue);
-        }
-
-        public Task<DValue<RecordValue>> UpdateField(string name, FormulaValue value)
```

Comment:
>UpdateField [](http://example.com/codeflow?start=41&length=11)

Other functions (GetFields, GetField) have both an async and a sync version; UpdateField doesn't. Any reason for that?

Reply:
Good point.